src/skmultiflow/drift_detection/hddm_w.py

```diff
@@ -30,7 +32,7 @@ class HDDM_W(BaseDriftDetector):
     References
     ----------
     .. [1] Frías-Blanco I, del Campo-Ávila J, Ramos-Jimenez G, et al.
-       Online and non-parametric drift detection methods based on Hoeffding’s bounds[J].
+       Online and non-parametric drift detection methods based on Hoeffding’s bounds.
```

Comment:
The same here

Reply:
This is the paper's title so we can not change it.